eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/RatifySpec.hs

```diff
@@ -198,86 +219,132 @@ spoAndCCVotingSpec = do
+          (drep, _, _) <- setupSingleDRep 1_000_000_000
+          SJust committee <- getCommittee
+          committeeThreshold committee `shouldBe` 0 %! 1
+          Map.size (committeeMembers committee) `shouldBe` 2
+          forM_ (Map.keys $ committeeMembers committee) ccShouldNotBeExpired
+          (proposal, newConstitution) <- mkConstitutionProposal SNothing
+          gaiConstitution <- submitProposal proposal
+          submitYesVote_ (DRepVoter drep) gaiConstitution
+          mapM_ registerCommitteeHotKey (Map.keys $ committeeMembers committee)
+          passNEpochs 2
+          getConstitution `shouldReturn` newConstitution
+      it
+        "Constitution can be changed regardless of active committee votes"
+        $ whenPostBootstrap
+        $ do
+          modifyPParams $ \pp ->
+            pp
+              & ppCommitteeMinSizeL .~ 2
+          modifyCommittee $ fmap (committeeThresholdL .~ 0 %! 1)
+          (drep, _, _) <- setupSingleDRep 1_000_000_000
+          SJust committee <- getCommittee
+          committeeThreshold committee `shouldBe` 0 %! 1
+          Map.size (committeeMembers committee) `shouldBe` 2
+          forM_ (Map.keys $ committeeMembers committee) ccShouldNotBeExpired
+          (proposal, newConstitution) <- mkConstitutionProposal SNothing
+          gaiConstitution <- submitProposal proposal
+          submitYesVote_ (DRepVoter drep) gaiConstitution
+          hotKeys <- mapM registerCommitteeHotKey (Map.keys $ committeeMembers committee)
+          forM_ hotKeys $ \c ->
+            oneof
+              [ return ()
+              , submitYesVote_ (CommitteeVoter c) gaiConstitution
+              , submitVote_ VoteNo (CommitteeVoter c) gaiConstitution
+              ]
+          passNEpochs 2
+          getConstitution `shouldReturn` newConstitution
+    describe "When min size is 0" $ do
+      it
+        "Constitution can be changed if the commitee is inactive but has some active members"
+        . whenPostBootstrap
+        $ do
+          modifyPParams $ \pp ->
+            pp
+              & ppDRepVotingThresholdsL . dvtUpdateToConstitutionL .~ (0 %! 1)
+              & ppCommitteeMinSizeL .~ 0
+              & ppCommitteeMaxTermLengthL .~ EpochInterval 50
+          void $ setupActiveInactiveCCMembers 1 1 (0 %! 1)
+          newConstitution <- arbitrary
+          constitutionActionId <- submitGovAction $ NewConstitution SNothing newConstitution
+          logRatificationChecks constitutionActionId
+          passNEpochs 2
+          getConstitution `shouldReturn` newConstitution
+      it
+        "Constitution can be changed if there are no active members"
+        . whenPostBootstrap
+        $ do
+          modifyPParams $ \pp ->
+            pp
+              & ppCommitteeMinSizeL .~ 0
+              & ppDRepVotingThresholdsL . dvtUpdateToConstitutionL .~ (0 %! 1)
+          modifyCommittee $ fmap (committeeThresholdL .~ 0 %! 1)
+          SJust committee <- getCommittee
```

Comment:
I think there should also be a check that none of the members have an active hot key. I can never remember what state the CC is in by default, so I think it's better to make it explicit.